apps/webapp/app/routes/_app.orgs.$organizationSlug_.projects.new/route.tsx

```diff
@@ -1,24 +1,59 @@
 import { conform, useForm } from "@conform-to/react";
```

Comment:
It's now possible to have an Org with no projects. So we've moved the new project page outside of the org route.

If you have zero projects there's a panel explaining why you need to add a project. When we allow deleting orgs we can have a link to that page in the callout too.